app/workers/updatemail_worker.rb

```diff
@@ -0,0 +1,8 @@
+class UpdatemailWorker
+    include Sidekiq::Worker
+   
+    def perform(interview_id, time)
+        # @interview = Interview.find(interview_id)
+        ParticipantMailer.update_mail(interview_id,time).deliver_now
```

Comment:
always try to use dleiver_later in mails and perform_later in jobs

Reply:
performed the changes in - 64a01bd